### PR TITLE
Heist gems alt quals

### PR DESCRIPTION
**Pull request by uilman**

### CODE REVIEW

File: Data/3_0/SkillStatMap.lua [hunk 1]
```diff
@@ -1351,5 +1475,7 @@ return {
 ["snipe_triggered_skill_ailment_damage_+%_final_per_stage"] = {
 	mod("Damage", "MORE", nil, ModFlag.Ailment, 0, { type = "Multiplier", var = "SnipeStage" }),
 },
-
+["withered_on_hit_chance_%"] = {
+	flag("Condition:CanWither"),
+},
 }
```
Comment (ppoelzl) [inline Data/3_0/SkillStatMap.lua:1481]: Please keep the newline at the end of the file.

File: Data/3_0/Skills/act_str.lua [hunk 2]
```diff
@@ -1038,17 +1038,17 @@ skills["BloodSandStance"] = {
...
-		["projectile_speed_+%_in_sand_stance"] = {
-			mod("Damage", "INC", nil, bit.bor(ModFlag.Attack, ModFlag.Projectile), 0, { type = "GlobalEffect", effectType = "Buff", effectName = "Blood", effectCond = "BloodStance" })
-		},
-		["spell_area_damage_+%_in_blood_stance"] = {
-			mod("AreaOfEffect", "INC", nil, bit.bor(ModFlag.Area, ModFlag.Spell), 0, { type = "GlobalEffect", effectType = "Buff", effectName = "Sand", effectCond = "SandStance" })
-		},
-		["spell_area_of_effect_+%_in_sand_stance"] = {
-			mod("Damage", "INC", nil, bit.bor(ModFlag.Area, ModFlag.Spell), 0, { type = "GlobalEffect", effectType = "Buff", effectName = "Blood", effectCond = "BloodStance" })
-		},
+        mod("Damage", "INC", nil, bit.bor(ModFlag.Attack, ModFlag.Projectile), 0, { type = "GlobalEffect", effectType = "Buff", effectName = "Blood", effectCond = "BloodStance"})
```
Comment (ppoelzl) [inline Data/3_0/Skills/act_str.lua:1041]: To me, this seems to be under-indented.

File: Export/Skills/act_str.txt [hunk 3]
```diff
@@ -206,17 +206,17 @@ local skills, mod, flag, skill = ...
...
-		["projectile_speed_+%_in_sand_stance"] = {
-			mod("Damage", "INC", nil, bit.bor(ModFlag.Attack, ModFlag.Projectile), 0, { type = "GlobalEffect", effectType = "Buff", effectName = "Blood", effectCond = "BloodStance" })
-		},
-		["spell_area_damage_+%_in_blood_stance"] = {
-			mod("AreaOfEffect", "INC", nil, bit.bor(ModFlag.Area, ModFlag.Spell), 0, { type = "GlobalEffect", effectType = "Buff", effectName = "Sand", effectCond = "SandStance" })
-		},
-		["spell_area_of_effect_+%_in_sand_stance"] = {
-			mod("Damage", "INC", nil, bit.bor(ModFlag.Area, ModFlag.Spell), 0, { type = "GlobalEffect", effectType = "Buff", effectName = "Blood", effectCond = "BloodStance" })
-		},
+        mod("Damage", "INC", nil, bit.bor(ModFlag.Attack, ModFlag.Projectile), 0, { type = "GlobalEffect", effectType = "Buff", effectName = "Blood", effectCond = "BloodStance"})
```
Comment (ppoelzl) [inline Export/Skills/act_str.txt:209]: This seems to be under-indented as well.

**Review by ppoelzl — CHANGES_REQUESTED** [review]

File: Export/Skills/act_str.txt (hunk 3 above)
Comment (uilman) [reply Export/Skills/act_str.txt:209]: Yea it happened when I handled the conflicts in github strangely

**Review by LocalIdentity — APPROVED** [review]

**Review by ppoelzl — APPROVED** [review]
LGTM.

**Pull request merged**